### PR TITLE
feat(nexus): online degraded child

### PR DESCRIPTION
**Pull request by dsavitskiy**

### CODE REVIEW

**Comment by dsavitskiy** [comment]
bors try

**Comment by dsavitskiy** [comment]
bors merge

File: io-engine/src/bdev/nexus/nexus_child.rs [hunk 1]
```diff
@@ -44,10 +44,8 @@ use spdk_rs::{
 #[derive(Debug, Snafu)]
 #[snafu(context(suffix(false)))]
 pub enum ChildError {
-    #[snafu(display("Child is not offline"))]
-    ChildNotOffline {},
-    #[snafu(display("Child is not closed"))]
-    ChildNotClosed {},
+    #[snafu(display("Child is permanently faulted"))]
+    PermanemtlyFaulted {},
```
Comment (tiagolobocastro) [inline io-engine/src/bdev/nexus/nexus_child.rs]: typo: `PermanentlyFaulted`

**Review by tiagolobocastro — APPROVED** [review]

**Review by Abhinandan-Purkait — APPROVED** [review]

File: io-engine/src/bdev/nexus/nexus_child.rs (hunk 1 above)
Comment (dsavitskiy) [reply io-engine/src/bdev/nexus/nexus_child.rs]: ok will fix

**Comment by dsavitskiy** [comment]
bors merge

**Comment by dsavitskiy** [comment]
bors merge

**Review by mtzaurus — APPROVED** [review]

**Comment by dsavitskiy** [comment]
bors merge

**Comment by dsavitskiy** [comment]
bors merge

**Comment by dsavitskiy** [comment]
bors merge

**Comment by dsavitskiy** [comment]
bors merge

**Pull request merged**